editor/project_manager/project_dialog.cpp

```diff
@@ -245,6 +245,36 @@ void ProjectDialog::_validate_path() {
 			}
 		}
 	}
+
+	// Check if the target path is a subdirectory of original when duplicating
+	if (mode == MODE_DUPLICATE) {
+		String base_path = original_project_path;
+		String duplicate_target = target_path;
+
+		// Ensure the paths end with a slash
+		if (!base_path.ends_with("/")) {
+			base_path += "/";
+		}
+
+		if (!duplicate_target.ends_with("/")) {
+			duplicate_target += "/";
+		}
+
+#ifdef WINDOWS_ENABLED
+		// On Windows, make both lowercase for case-insensitive comparison
```

Comment:
Not only Windows is case-insensitive, you should use `DirAccess.is_case_sensitive()` instead.

Reply:
Don't push changes without compiling locally .-.
`is_case_sensitive()` takes an argument.